ckanext/saml2/plugin.py

```diff
@@ -211,6 +213,55 @@ def saml2_user_delete(context, data_dict):
     ckan_user_delete(context, data_dict)
 
 
+def saml2_check_for_user_update(id):
+    c = p.toolkit.c
+    user_info = saml2_get_user_info(id)
+    if user_info is not None:
+        c.allow_user_changes = p.toolkit.asbool(
+            config.get('ckan.saml2.allow_user_changes', False))
+        c.is_allow_update = user_info[0].allow_update
+    else:
+        c.allow_user_changes = c.is_allow_update = None
+
+
+def saml2_user_update(context, data_dict):
+    id = logic.get_or_bust(data_dict, 'id')
+    c = p.toolkit.c
+    saml2_check_for_user_update(id)
+    name_id = saml2_get_user_name_id(id)
+    if name_id is not None:
+        if c.allow_user_changes:
+            allow_update_params = data_dict.get('allow_update')
+            if allow_update_params is not None:
+                allow_update_params = p.toolkit.asbool(data_dict.get('allow_update'))
```

Comment:
variable name ends with `s` - does this mean, that you store multiple values there